examples/device-dashboard/Makefile

```diff
@@ -9,13 +9,11 @@ CFLAGS = -W -Wall -Wextra -g -I.        # Build options
 CFLAGS_MONGOOSE +=
 
 ifeq ($(OS),Windows_NT)         # Windows settings. Assume MinGW compiler. To use VC: make CC=cl CFLAGS=/MD OUT=/Feprog.exe
-  PROG ?= example.exe           # Use .exe suffix for the binary
+  PROG = example.exe            # Use .exe suffix for the binary
```

Comment:
Besides not letting the user change the name, this implies changing all other tutorials so all are consistent and we don't diverge again having lots of different ways of doing the same
Is it really necessary ?